packages/react-vapor/src/components/toast/examples/ToastConnectedExamples.1.use.md

```diff
@@ -0,0 +1,35 @@
+## Guidelines
+
+### Content
+
+-   Only include information that is relevant to the performed action.
+
+-   Toasts may contain action buttons. Try to include no more than one action button besides the one to dismiss the Toast.
+
+### Behavior
+
+-   Only one Toast should be displayed at a time. If two consecutive Toasts have to be shown, the first one should be removed before the second one is shown.
```

Comment:
Is this right? Because our error toast don't close automatically. If a new one open I still don't think that the first one should close